test/unittest/unittest_nntrainer_tensor_neon_fp16.cpp

```diff
@@ -384,6 +384,47 @@ TEST(nntrainer_Tensor, max_abs) {
   EXPECT_NEAR(result_neon, result_fp32, epsilon);
 }
 
+TEST(nntrainer_Tensor, max_abs_768) {
```

Comment:
Please add test cases for isamax, verifying its results, where:
1. N == UINT16_MAX
2. N == UINT16_MAX - 1
3. N >> UINT16_MAX